doc/tutorial/quick-start.md

```diff
@@ -0,0 +1,34 @@
+# Quick start
```

Comment:
I advise choosing a different title than "Quick start" for the following reasons:

- A quick start is typically a how-to guide and not a tutorial. 
- Tutorials are rarely "quick" about anything; it's less about being quick and more about giving the user a valuable learning experience. 
- "Quick start" as a title does not give the user any insight about what the tutorial is going to cover.

Based on the "What you'll do" section, I suggest the title "Initialise and run the GitHub image builder for the first time". I'm eager to hear your thoughts :)

Reply:
I've renamed it to `Build your first image` - since we're initializing and running the build in this tutorial :D

Reply:
That sounds great to me, thanks!